xml/System.Security.Cryptography/SHA256Managed.xml

```diff
@@ -398,10 +398,10 @@ This method is called by the public <xref:System.Security.Cryptography.HashAlgor
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">The buffer to receive the hash value.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written into <paramref name="destination"/>. This parameter is treated as uninitialized.</param>
+        <summary>Attempts to finalize the hash computation after the last data is processed by the hash algorithm.</summary>
+        <returns><see langword="true"/> if <paramref name="destination"/> is long enough to receive the hash value; otherwise, <see langword="false"/>.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true"/> if <paramref name="destination"/> is long enough to receive the hash value; otherwise, <see langword="false"/>.</returns>
```